io/src/pcd_io.cpp

```diff
@@ -447,6 +451,14 @@ pcl::PCDReader::readBodyASCII (std::istream &fs, pcl::PCLPointCloud2 &cloud, int
       boost::trim (line);
       boost::split (st, line, boost::is_any_of ("\t\r "), boost::token_compress_on);
 
+      if (st.size () != elems_per_line) // If this is not checked, an exception might occur when accessing st
+      {
+        PCL_WARN ("[pcl::PCDReader::readBodyASCII] Possibly misformed PCD file: line has %zu elements, but should have %u! (point %u)\n",
```

Comment:
I think malformed is a better word in this case

Reply:
Rewording the last sentence might also help: `Point number %u has %zu elements, but should have %u` or similar